firmware/library/L2_HAL/memory/sddriver_test.cpp

```diff
@@ -0,0 +1,15 @@
+// Tests for the SD Card Driver Class
+#include "L3_HAL/sddriver.hpp"
+#include "L5_Testing/testing_frameworks.hpp"
+
+TEST_CASE("Testing SD Card Driver Class", "[sd card driver]")
+{
+    using fakeit::Fake;
+    using fakeit::Mock;
+    using fakeit::Spy;
+    using fakeit::Verify;
+    using fakeit::When;
+
+    Mock<SspInterface> mock_cs;     // CS:      SSP Chip Select
```

Comment:
Remove this line.

Reply:
Which line? The "using" forwarded lines or the mock_cs?